rules.bzl

```diff
@@ -120,6 +124,7 @@ UBUNTU22_04_IMAGE = ":".join([UBUNTU22_04_REPOSITORY, "latest"])
 def buildbuddy(
         name,
         container_image = None,
+        nonroot_workspace = True,
```

Comment:
I default to `True` because the docs say the Docker images are expected to run with the `buildbuddy` user... if this should default to `False` let me know!

Reply:
Thanks for sending this change! I do think that we could benefit from simplifying the non-root setup. I think ideally though, we could reduce the need for some of this manual configuration. I will talk with the team internally about this, but I think we could either make all workspaces writable by non-root by default, or have nonroot-workspace default to true when dockerUser is a non-root user.

Reply:
@bduffany agreed, having it "just work" would be even better than having the default in the toolchain. And piggybacking on `dockerUser` makes sense. But then I'd make it explicit in the toolchain because right now, it seems to be an implementation detail that's just "barely documented" in the docs... it took me a while to re-read it all and understand the user part + the nonroot part, etc.

I think the toolchain should set the `dockerUser` to `buildbuddy` by default and that would signal people using it that it could be changed to any other non-`root` (or `root`) user.

What do you think?